hydra-filters/src/main/java/com/addthis/hydra/data/filter/value/ValueFilterHash.java

```diff
@@ -99,6 +101,10 @@ public ValueObject filterValue(ValueObject value) {
                 } catch (NoSuchAlgorithmException e) {
                     // ignore
                 }
+            case 5:
+                StringAndByteHashFunction mmh = new Murmur3HashFunction();
```

Comment:
StringAndByteHashFunction is a interface and implemented by some classes. To make methods in Murmur3HashFunction static, we need also make methods in its parent also static, but interface can't have static method in java7. In java8, we can have static methods in an interface, but  we can’t override them in the implementation classes.  If you want to make all method calls in ValueFilterHash static, need to change structure of related classes using pattern (eg factory pattern).